www/grpc/proto/network.proto

```diff
@@ -71,10 +61,14 @@ message GetNodeInfoResponse {
   repeated string local_addrs = 8;
   // List of protocols supported by the node.
   repeated string protocols = 9;
+  // If the blocks are subject to pruning.
+  bool is_pruned = 10;
+  // Lowest-height block stored (only present if pruning is enabled)
+  int32 pruning_height = 11;
   // Clock offset of the node.
-  double clock_offset = 13;
```

Comment:
@kehiy don't change the old numbers. Keep them the same for backward compatibility. I know we changed them in the past, but now more users are using Pactus, and we can't ask them to upgrade their applications all the time.

Reply:
@b00f Fixed, now we didn't changes the numbers, only added some of them. anything must be backward compatible now.